ko/error-code.md

```diff
@@ -1,184 +1,154 @@
-| false        | -9999      | Internal error.                                                                                                            |
+| 카테고리  | 성공 여부 | 결과 코드 | 결과 코드 메시지                                                         | API 응답 메시지                                                                                                      |
+|-------|-------|-------|-------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------|
+| 공통    | true  | 0     | 성공                                                                | success                                                                                                         |
+| 공통    | true  | 1000  | 일부 요청이 성공했습니다.                                                    | Some of your requests are successful.                                                                           |
+| 공통    | false | -1000 | 유효하지 않은 appKey입니다.                                                | Invalid appKey.                                                                                                 |
+| 공통    | false | -1001 | 서비스가 존재하지 않습니다.                                                   | Service is not exist                                                                                            |
+| 공통    | false | -1002 | 서비스가 비활성화되어 있습니다.                                                 | Service is disabled                                                                                             |
+| 공통    | false | -1003 | 유효하지 않은 서비스 파라미터입니다.                                              | Invalid service parameter                                                                                       |
+| 공통    | false | -1004 | 유효하지 않은 secretKey입니다.                                             | Invalid secretKey.                                                                                              |
+| 공통    | false | -1005 | 서비스가 일시적으로 차단되었습니다.                                               | Service is temporarily blocked                                                                                  |
+| 공통    | false | -1007 | 서비스의 'useYn' 필드는 필수입니다.                                           | The 'useYn' field of the service is required                                                                    |
+| 공통    | false | -2000 | 날짜 형식이 유효하지 않습니다.                                                 | Date format is invalid. {}: {}, Only supported {}                                                               |
+| 공통    | false | -2028 | 행 수 제한을 초과했습니다. 최대 50,000행까지 처리할 수 있습니다.                          | Limit of rows is exceeded. It can handle up to 50,000 rows.                                                     |
```

Comment:
```suggestion
| 공통    | false | -2028 | 행 개수 제한을 초과했습니다. 최대 50,000행까지 처리할 수 있습니다.                          | Limit of rows is exceeded. It can handle up to 50,000 rows.                                                     |
```